mods/gen3/items.js

```diff
@@ -215,6 +217,7 @@ exports.BattleItems = {
 		onEat: function (pokemon) {
 			this.heal(30);
 		},
+		desc: "The holder heals 30 HP.",
```

Comment:
Why did you remove this? Clearly I meant "when does the holder heal 30 HP?" You can't just say "heals 30 HP", that doesn't mean anything. It's a Berry, it has an activation condition.